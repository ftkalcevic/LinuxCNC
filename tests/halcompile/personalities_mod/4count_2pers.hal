# personalities are used modulo the number of
# halcompiled personalities

# 4 count, 2 personalities
loadrt logic    count=4 personality=0x102,0x204
loadrt lincurve count=4 personality=2,3
loadrt bitslice count=4 personality=2,3

show pin
show parameter
