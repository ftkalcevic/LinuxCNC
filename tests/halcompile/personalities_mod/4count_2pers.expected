bitslice u32   IN 0 bitslice.0.in
bitslice bit   OUT FALSE bitslice.0.out-00
bitslice bit   OUT FALSE bitslice.0.out-01
bitslice s32   OUT 0 bitslice.0.time
bitslice u32   IN 0 bitslice.1.in
bitslice bit   OUT FALSE bitslice.1.out-00
bitslice bit   OUT FALSE bitslice.1.out-01
bitslice bit   OUT FALSE bitslice.1.out-02
bitslice s32   OUT 0 bitslice.1.time
bitslice u32   IN 0 bitslice.2.in
bitslice bit   OUT FALSE bitslice.2.out-00
bitslice bit   OUT FALSE bitslice.2.out-01
bitslice s32   OUT 0 bitslice.2.time
bitslice u32   IN 0 bitslice.3.in
bitslice bit   OUT FALSE bitslice.3.out-00
bitslice bit   OUT FALSE bitslice.3.out-01
bitslice bit   OUT FALSE bitslice.3.out-02
bitslice s32   OUT 0 bitslice.3.time
lincurve float IN 0 lincurve.0.in
lincurve float OUT 0 lincurve.0.out
lincurve float I/O 0 lincurve.0.out-io
lincurve s32   OUT 0 lincurve.0.time
lincurve float IN 0 lincurve.1.in
lincurve float OUT 0 lincurve.1.out
lincurve float I/O 0 lincurve.1.out-io
lincurve s32   OUT 0 lincurve.1.time
lincurve float IN 0 lincurve.2.in
lincurve float OUT 0 lincurve.2.out
lincurve float I/O 0 lincurve.2.out-io
lincurve s32   OUT 0 lincurve.2.time
lincurve float IN 0 lincurve.3.in
lincurve float OUT 0 lincurve.3.out
lincurve float I/O 0 lincurve.3.out-io
lincurve s32   OUT 0 lincurve.3.time
logic bit   OUT FALSE logic.0.and
logic bit   IN FALSE logic.0.in-00
logic bit   IN FALSE logic.0.in-01
logic s32   OUT 0 logic.0.time
logic bit   IN FALSE logic.1.in-00
logic bit   IN FALSE logic.1.in-01
logic bit   IN FALSE logic.1.in-02
logic bit   IN FALSE logic.1.in-03
logic bit   OUT FALSE logic.1.or
logic s32   OUT 0 logic.1.time
logic bit   OUT FALSE logic.2.and
logic bit   IN FALSE logic.2.in-00
logic bit   IN FALSE logic.2.in-01
logic s32   OUT 0 logic.2.time
logic bit   IN FALSE logic.3.in-00
logic bit   IN FALSE logic.3.in-01
logic bit   IN FALSE logic.3.in-02
logic bit   IN FALSE logic.3.in-03
logic bit   OUT FALSE logic.3.or
logic s32   OUT 0 logic.3.time

bitslice s32   RW 0 bitslice.0.tmax
bitslice bit   RO FALSE bitslice.0.tmax-increased
bitslice s32   RW 0 bitslice.1.tmax
bitslice bit   RO FALSE bitslice.1.tmax-increased
bitslice s32   RW 0 bitslice.2.tmax
bitslice bit   RO FALSE bitslice.2.tmax-increased
bitslice s32   RW 0 bitslice.3.tmax
bitslice bit   RO FALSE bitslice.3.tmax-increased
lincurve s32   RW 0 lincurve.0.tmax
lincurve bit   RO FALSE lincurve.0.tmax-increased
lincurve float RW 0 lincurve.0.x-val-00
lincurve float RW 0 lincurve.0.x-val-01
lincurve float RW 0 lincurve.0.y-val-00
lincurve float RW 0 lincurve.0.y-val-01
lincurve s32   RW 0 lincurve.1.tmax
lincurve bit   RO FALSE lincurve.1.tmax-increased
lincurve float RW 0 lincurve.1.x-val-00
lincurve float RW 0 lincurve.1.x-val-01
lincurve float RW 0 lincurve.1.x-val-02
lincurve float RW 0 lincurve.1.y-val-00
lincurve float RW 0 lincurve.1.y-val-01
lincurve float RW 0 lincurve.1.y-val-02
lincurve s32   RW 0 lincurve.2.tmax
lincurve bit   RO FALSE lincurve.2.tmax-increased
lincurve float RW 0 lincurve.2.x-val-00
lincurve float RW 0 lincurve.2.x-val-01
lincurve float RW 0 lincurve.2.y-val-00
lincurve float RW 0 lincurve.2.y-val-01
lincurve s32   RW 0 lincurve.3.tmax
lincurve bit   RO FALSE lincurve.3.tmax-increased
lincurve float RW 0 lincurve.3.x-val-00
lincurve float RW 0 lincurve.3.x-val-01
lincurve float RW 0 lincurve.3.x-val-02
lincurve float RW 0 lincurve.3.y-val-00
lincurve float RW 0 lincurve.3.y-val-01
lincurve float RW 0 lincurve.3.y-val-02
logic s32   RW 0 logic.0.tmax
logic bit   RO FALSE logic.0.tmax-increased
logic s32   RW 0 logic.1.tmax
logic bit   RO FALSE logic.1.tmax-increased
logic s32   RW 0 logic.2.tmax
logic bit   RO FALSE logic.2.tmax-increased
logic s32   RW 0 logic.3.tmax
logic bit   RO FALSE logic.3.tmax-increased

